docs/tutorials/google/echoes.ipynb

```diff
@@ -0,0 +1,703 @@
+      "text/plain": [
+       "<Figure size 504x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light",
+      "tags": []
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Average data over trials.\n",
+    "avg_probs = np.average(probs, axis=0)\n",
+    "std_probs = np.std(probs, axis=0)\n",
+    "\n",
+    "# Plotting.\n",
+    "plt.figure(figsize=(7, 5))\n",
+    "\n",
+    "step = len(cycle_values)\n",
+    "stop = len(avg_probs) // step\n",
+    "for i in range(stop):\n",
+    "    plt.errorbar(\n",
+    "        x=cycle_values, \n",
+    "        y=avg_probs[i * step: (i + 1) * step], \n",
+    "        yerr=std_probs[i * step: (i + 1) * step],\n",
+    "        capsize=5,\n",
+    "        lw=2,\n",
+    "        label=f\"Qubit configuration {i}\"\n",
+    "    )\n",
+    "\n",
+    "plt.legend()\n",
+    "plt.ylabel(\"Survival probability\")\n",
+    "plt.xlabel(\"Cycle\")\n",
+    "plt.grid(\"on\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "0WGZ9T2q6YI1"
+   },
+   "source": [
+    "The initial point (at zero cycles) reflects readout error, and the decay rate reflects the gate error per cycle. We fit an exponential to each curve to determine the gate error per cycle below."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "H2QMj50-ODPc"
+   },
+   "source": [
+    "Note: To ensure good fit parameters are calculated, it is important to collect enough data such that the above curves reach their asymptote."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "id": "4AB4mkmQ6WkC"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Error/cycle on qubit configuration 0: 3.49%\n",
+      "Error/cycle on qubit configuration 1: 6.45%\n",
+      "Error/cycle on qubit configuration 2: 4.46%\n",
+      "Error/cycle on qubit configuration 3: 3.87%\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"Fit an exponential decay to the collected data.\"\"\"\n",
+    "from scipy.optimize import curve_fit\n",
+    "\n",
+    "def fit(cycle, a, f):\n",
+    "    return a * np.exp((f - 1.0) * cycle)\n",
+    "\n",
+    "\n",
+    "for i in range(stop):\n",
+    "    (a, f), _ = curve_fit(\n",
+    "        fit, \n",
+    "        xdata=cycle_values, \n",
+    "        ydata=avg_probs[i * step: (i + 1) * step],\n",
+    "    )\n",
+    "    print(f\"Error/cycle on qubit configuration {i}: {round((1 - f) * 100, 2)}%\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "tf-kNlevK12O"
+   },
+   "source": [
+    "Note: The definition of cycle here is different than that used in the calibration metrics."
```

Comment:
How so? Maybe a little explanation is worthwhile given that we're pointing it out.

Reply:
Agree - I think it's a factor of 2 but need to verify this. For now it serves as a (shallow) explanation why the per cycle error here is larger than the per cycle calibration error.